dracut/30ignition/ignition-setup-base.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Ignition (setup)
```

Comment:
A distinction between system and user setup would be good.

Reply:
Good catch, fixed.